### ISSUE TITLE
IntelliJ installs an old version of the plugin

### ISSUE THREAD

**Issue by arpadtamasi** [samebug/samebug-idea-plugin#7]
Details from @buchireddy:

Seems like the samebug version is 0.12.1+idea163.0 and I'm using Intellij 2017.1.5 (the latest one)

**Comment by arpadtamasi** [comment]
I deleted a few old versions from the repository, but cannot delete this one. 
This version says it works with IntelliJ 163+ while the latest one with 141+ so JetBrains thinks this is the latest.

https://plugins.jetbrains.com/plugin/8174-samebug/update/30826

**Comment by arpadtamasi** [comment]
I asked JetBrains to delete the outdated version.
https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000451090-Plugin-update-removal-request

**Comment by arpadtamasi** [comment]
Info from JetBrains: I've checked your plugin and there was a problem with latest updates that they were marked as compatible with Idea Ultimate only while they should be compatible with Idea Community too. I've fixed that and now version "0.15.1+idea" is listed in both Idea Ultimate and Idea Community. Does it solve your problem or you still want that update to be delete? I will investigate why compatibility problem has happened in the first place.
